test/localhub/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.9"
+services:
+  tests:
+    image: "${DOCKER_REPOSITORY}/localhub-test:${DOCKER_TAG}"
+    depends_on:
+      hubjs:
+        condition: service_healthy
+    environment:
+      - LOCALHUB_ADDR=http://hubjs:8080/graphql
+      - GRPC_SECRET_STORAGE_BACKEND_ADDR=http://backend-storage:50051
+  hubjs:
+    image: "${DOCKER_REPOSITORY}/hub-js:${DOCKER_TAG}"
+    restart: on-failure
+    ports:
+      - "8080:8080"
+    environment:
+      - APP_NEO4J_ENDPOINT=bolt://neo4j:7687
+      - NEO4JLABS_PLUGINS=["apoc"]
+      - APP_NEO4J_PASSWORD=okon
+      - APP_HUB_MODE=local
+      - APP_GRAPH_QL_ADDR=:8080
+    depends_on:
+      neo4j:
+        condition: service_healthy
+    healthcheck:
+      test: netstat -ltn | grep -c 8080
+      interval: 5s
+      timeout: 5s
+      retries: 10
+      start_period: 60s
+  neo4j:
+    image: "ghcr.io/capactio/neo4j:4.2.13-apoc"
+    restart: on-failure
+    ports:
+      - "7687:7687"
+      - "7474:7474"
+    environment:
+      - NEO4J_AUTH=neo4j/okon
+      - NEO4JLABS_PLUGINS=["apoc"]
+    healthcheck:
+      test: wget http://localhost:7474/browser -O -
+      interval: 10s
+      timeout: 10s
+      retries: 10
+      start_period: 40s
+  backend-storage:
+    image: "${DOCKER_REPOSITORY}/secret-storage-backend:${DOCKER_TAG}"
+    restart: on-failure
```

Comment:
Please add healtcheck as well 🙂

Reply:
Actually, I skipped adding health checks here deliberately because AFAIK there is no really good way of running the testing command on that image. It does not contain `wget`, `curl` so probably, if we really want to do this, we could do the wrapping and build the image here based on the original image by adding a "testing" tool.

Reply:
Alright, let's skip this - we can follow-up if we see any issues without this healthcheck 👍